overlord/state/notices.go

```diff
@@ -243,7 +259,12 @@ func (s *State) AddNotice(userID *uint32, noticeType NoticeType, key string, opt
 
 	now := options.Time
 	if now.IsZero() {
-		now = time.Now()
+		now = getTimeNow()
```

Comment:
`time.Now()` is already mock-able.
```suggestion
		now = timeNow()
```

Reply:
Good point. Changing everything.